File: Modul 1 Sem 1/C/PRAK104-2410817110005-MuhammadRizqanKarim.c
#include <stdio.h>

int main()
{
    int pA, pB, sA = 400000, sB = 350000;
    float discA = 0.13, discB = 0.21;

    printf("Harga sepatu A adalah %d\n", sA);
    printf("Harga sepatu B adalah %d\n", sB);

    pA = sA - (sA * discA);
    pB = sB - (sB * discB);

    printf("Sepatu A mendapat diskon %.0f%% sehingga harganya menjadi %d\n", discA*100, pA);
    printf("Sepatu B mendapat diskon %.0f%% sehingga harganya menjadi %d", discB*100, pB);

    return 0;
}

File: Modul 1 Sem 1/Phython/PRAK104-2410817110005-MuhammadRizqanKarim.py
sA = 400000 
sB = 350000
dcA = 13
dcB = 21

print(f"Variabel a bernilai {sA}")
print(f"Variabel b bernilai {sB}")

pA = sA - (sA * (dcA/100))
pB = sB - (sB * (dcB/100))

print(f"Sepatu A mendapat diskon {dcA}% sehingga harganya menjadi {pA:.0f}")
print(f"Sepatu B mendapat diskon {dcB}% sehingga harganya menjadi {pB:.0f}")
